extensions/blocks/tiled-gallery/index.js

```diff
@@ -133,10 +134,11 @@ export const icon = (
 export const settings = {
 	attributes: blockAttributes,
 	category: 'jetpack',
-	description: __(
-		"Display multiple images in an elegantly organized tiled layout. Serves images using Jetpack's fast global network of servers.",
-		'jetpack'
-	),
+	description:
+		__( 'Display multiple images in an elegantly organized tiled layout.', 'jetpack' ) +
+		( ! isAtomicSite() && ! isSimpleSite()
```

Comment:
Any specific reason why we wouldn't want to display this information to Atomic site owners? They do have access to Jetpack Settings where the CDN is introduced.

Reply:
Two reasons I can think of: 

1. Jetpack brand isn't really introduced to these customers at any point so it could be confusing
2. All their images are already served using "WordPress.com's fast global network of servers" so to customer this information doesn't make any difference and can be confusing.

I honestly expected us to manage .com business plans in such a way that CDN would _always_ be turned on by default without options to turn it off. :-)

Reply:
Atomic site owners get to activate and deactivate Photon just like regular Jetpack site owners, and they do have access to the Jetpack menu in wp-admin, so I think it'd be fine to use that vocabulary with them.

Reply:
Leaving it for someone else on Crew to chime in with their opinion on this. 👍

Reply:
I don't feel strongly either way; I left the text now for atomic and removing it for simple sites. I think we can agree about simple sites anyway. :-)

Reply:
@lancewillett can you help to flip a coin here? 😁 

We're thinking if to allow Atomic WordPress.com customers to see _"Serves images using Jetpack's fast global network of servers."_ sentence in Tiled gallery description:

<img width="298" alt="image" src="https://user-images.githubusercontent.com/87168/67549072-0f4ec700-f70c-11e9-8fa3-d077822f2e4e.png">